presto-docs/src/main/sphinx/release/release-0.222.rst

```diff
@@ -0,0 +1,44 @@
+=============
+Release 0.222
+=============
+
+General Changes
+---------------
+
+* Fix incorrect results when dividing certain decimal numbers (:pr:`12930`).
+* Fix planning failure for certain join queries caused by invalid distribution
+  type (:pr:`12354`).
+* Add support for automatic retries for failed buckets when using grouped
+  execution for certain queries when using the Hive connector. This can be
+  enabled with the ``recoverable_grouped_execution`` session property (:issue:`12124`).
+* Add support for grouped execution for queries with no joins or aggregations.
```

Comment:
Same thing about putting properties first and also about whether there is a corresponding configuration property. 

```
Add session property ``grouped_execution_for_eligible_table_scans`` [and configuration property...] to enable grouped execution for queries with no joins or aggregations (:pr:`12934`).
```